### PR TITLE
Add simple issue templates

### PR DESCRIPTION
**Pull request by rasbt**
Adds simple issue templates to better categorize the issues and collect minimal info such as the LitGPT version, which will help with debugging.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
+name: Bug Report
```
Comment (Andrei-Aksionov) [inline .github/ISSUE_TEMPLATE/bug-report.yaml:1]: I'd recommend adding a note to have the latest version of LitGPT and up-to-date dependencies before submitting the issue.
Ideally to create a clean venv and install everything there with the command from README.

File: .github/ISSUE_TEMPLATE/bug-report.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+description: Report errors related to LitGPT
+title: "Description"
+labels: bug
+assignees: rasbt
```
Comment (rasbt) [inline .github/ISSUE_TEMPLATE/bug-report.yaml:5]:
```suggestion
```

File: .github/ISSUE_TEMPLATE/bug-report.yaml (hunk 1 above)
Comment (Andrei-Aksionov) [reply .github/ISSUE_TEMPLATE/bug-report.yaml:1]: Makes sense.
Let's start with something simple.

Comment (rasbt) [reply .github/ISSUE_TEMPLATE/bug-report.yaml:1]: Thanks for the feedback though!

**Pull request merged**